scripts/release.ts

```diff
@@ -181,10 +181,23 @@ async function main() {
     $(`git clone ${urlBase}tensorflow/${phase.repo} ${dir} --depth=1`);
     shell.cd(dir);
   } else {
-    // Publishing tfjs, clone the release branch.
-    $(`git clone -b ${releaseBranch} ${urlBase}tensorflow/tfjs ${
-        dir} --depth=1`);
-    shell.cd(dir);
+    // Publishing tfjs, clone the release branch, if branch doesn't exist,
```

Comment:
this looks good. just curious, will we break things into release units soon?

Reply:
> this looks good. just curious, will we break things into release units soon?
Hi Nikhil, the script is ready, so in next release, it will be a release unit.